.gitignore

```diff
@@ -2,3 +2,4 @@ Gemfile.lock
 gemfiles/.bundle
 pkg/
 .rspec
+.idea
```

Comment:
I prefer to keep user/system specific things out of the project `gitignore`, would you take this out?

Reply:
ok, i'll fix after main fix accepting